chain/cosmos/cosmos_chain.go

```diff
@@ -506,13 +506,15 @@ func (c *CosmosChain) GetGasFeesInNativeDenom(gasPaid int64) int64 {
 	return int64(fees)
 }
 
-func (c *CosmosChain) UpgradeVersion(ctx context.Context, cli *client.Client, version string) {
+func (c *CosmosChain) UpgradeVersion(ctx context.Context, cli *client.Client, ContainerRepo, version string) {
```

Comment:
var name should be lowercase `containerRepo`

Reply:
👍